### PR TITLE
Registered the AKivymd classes in Kv language

### PR DESCRIPTION
**Pull request by kengoon**
I registered all the AKivymd classes in Kv language, so no need to import it through python file again. The only import needed is the "import akivymd" in python file or "#:import akivymd akivymd" in the root kv file. So anyone can use the AKivmd classes directly in the kv language without any other import statement after the initial "import akivymd" statement in python file or kivy file. 

I also modified the codes in the demo app

### CODE REVIEW

File: demo/screens/bottomnavigation.py [hunk 1]
```diff
@@ -18,14 +17,15 @@
...
         AKBottomNavigation:
             items: root.bottomnavigation_items
+
 '''
 )
 
 class BottomNavigation(Screen):
 
     bottomnavigation_items=[
-        {'icon':'android' , 'text': 'android' , 'on_release': lambda x: toast('android')},
```
Comment (quitegreensky) [inline demo/screens/bottomnavigation.py:27]: What has this to do with the main purpose?

File: demo/screens/silverappbar.py [hunk 2]
```diff
@@ -43,4 +42,4 @@ def on_enter(self, *args):
             self.ids.content.add_widget(OneLineListItem(text='Item %d'%x))
 
     def on_leave(self, *args):
-        self.ids.content.clear_widgets()
\ No newline at end of file
+        self.ids.content.clear_widgets()
```
Comment (quitegreensky) [inline demo/screens/silverappbar.py:45]: ??

File: demo/screens/selectionlist.py [hunk 3]
```diff
@@ -60,4 +60,4 @@ def clear_selected(self):
         return self.ids.selectionlist.clear_selection()
 
     def select_all(self):
-        return self.ids.selectionlist.select_all()
```
Comment (quitegreensky) [inline demo/screens/selectionlist.py:63]: ??

File: demo/screens/onboarding.py [hunk 4]
```diff
@@ -82,4 +81,4 @@
 class Onboarding(Screen):
 
     def finish_callback(self):
-        toast('Finish callback')
```
Comment (quitegreensky) [inline demo/screens/onboarding.py:85]: ??

File: demo/screens/imageviewer.py [hunk 5]
```diff
@@ -33,4 +33,4 @@ def __init__(self, **kw):
             ))
 
     def open(self):
-        self.viewer.open()
```
Comment (quitegreensky) [inline demo/screens/imageviewer.py:36]: ??

File: demo/screens/dataloader.py [hunk 6]
```diff
@@ -125,4 +123,4 @@ def clear_data(self):
         self.ids.user1.website= ''
 
 class Loadercard(MDCard):
-    pass
```
Comment (quitegreensky) [inline demo/screens/dataloader.py:128]: ??

File: demo/screens/bottomappbar.py [hunk 7]
```diff
@@ -40,4 +39,4 @@
 class BottomAppbar(Screen):
 
     def toast(self,x):
-        return toast(x)
```
Comment (quitegreensky) [inline demo/screens/bottomappbar.py:43]: ??

File: demo/screens/badgelayout.py [hunk 8]
```diff
@@ -75,4 +74,4 @@
 )
 
 class BadgeLayout(Screen):
-    pass 
\ No newline at end of file
+    pass
```
Comment (quitegreensky) [inline demo/screens/badgelayout.py:77]: ??

File: demo/screens/addwidget.py [hunk 9]
```diff
@@ -37,4 +37,4 @@ def update(self, *args):
         self.ids.list.items= items
     
     def on_leave(self): 
-        self.ids.list.clear_widgets()
```
Comment (quitegreensky) [inline demo/screens/addwidget.py:40]: ??

**Review by quitegreensky — COMMENTED** [review]
Thanks for contributing. Could you please make this a little cleaner and cancel unnecessary changes? also python cache files? 
Then send your PR to `dev` branch

File: demo/screens/bottomnavigation.py (hunk 1 above)
Comment (kengoon) [reply demo/screens/bottomnavigation.py:27]: sorry that was a my mistake, I never intended to post that too

**Comment by quitegreensky** [comment]
I merged your first two commits because i want to leave the demo app as it is for now. Thanks 🎖️

**Comment by kengoon** [comment]
> I merged your first two commits because i want to leave the demo app as it is for now. Thanks medal_military

ok

**Pull request closed without merging**